xtendActiveAnnotationExamples fails in xtext-reference-projects

https://github.com/xtext/xtext-reference-projects/actions/runs/12079364481/job/33685464573

```
com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: java.lang.NoClassDefFoundError: org/eclipse/xtext/testing/TemporaryFolder
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2083)
	at com.google.common.cache.LocalCache.get(LocalCache.java:4017)
	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:4040)
	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4989)
	at com.google.common.cache.LocalCache$LocalLoadingCache.getUnchecked(LocalCache.java:4996)
	at com.google.inject.internal.FailableCache.get(FailableCache.java:61)
	at com.google.inject.internal.ConstructorInjectorStore.get(ConstructorInjectorStore.java:53)
	at com.google.inject.internal.ConstructorBindingImpl.initialize(ConstructorBindingImpl.java:152)
	at com.google.inject.internal.InjectorImpl.initializeJitBinding(InjectorImpl.java:607)
	at com.google.inject.internal.InjectorImpl.createJustInTimeBinding(InjectorImpl.java:990)
	at com.google.inject.internal.InjectorImpl.createJustInTimeBindingRecursive(InjectorImpl.java:902)
	at com.google.inject.internal.InjectorImpl.getJustInTimeBinding(InjectorImpl.java:302)
	at com.google.inject.internal.InjectorImpl.getBindingOrThrow(InjectorImpl.java:225)
	at com.google.inject.internal.InjectorImpl.getProviderOrThrow(InjectorImpl.java:1140)
	at com.google.inject.internal.InjectorImpl.getProvider(InjectorImpl.java:1169)
	at com.google.inject.internal.InjectorImpl.getProvider(InjectorImpl.java:1134)
	at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1186)
	at org.eclipse.xtend.core.compiler.batch.XtendCompilerTester.newXtendCompilerTester(XtendCompilerTester.java:56)
	at extract.ExtractTest.<init>(ExtractTest.java:28)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:250)
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:260)
	at org.junit.runners.BlockJUnit4ClassRunner$2.runReflectiveCall(BlockJUnit4ClassRunner.java:309)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:757)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
Caused by: com.google.common.util.concurrent.ExecutionError: java.lang.NoClassDefFoundError: org/eclipse/xtext/testing/TemporaryFolder
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2083)
	at com.google.common.cache.LocalCache.get(LocalCache.java:4017)
	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:4040)
	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4989)
	at com.google.common.cache.LocalCache$LocalLoadingCache.getUnchecked(LocalCache.java:4996)
	at com.google.inject.internal.FailableCache.get(FailableCache.java:61)
	at com.google.inject.internal.MembersInjectorStore.get(MembersInjectorStore.java:69)
	at com.google.inject.internal.ConstructorInjectorStore.createConstructor(ConstructorInjectorStore.java:79)
	at com.google.inject.internal.ConstructorInjectorStore.access$000(ConstructorInjectorStore.java:29)
	at com.google.inject.internal.ConstructorInjectorStore$1.create(ConstructorInjectorStore.java:37)
	at com.google.inject.internal.ConstructorInjectorStore$1.create(ConstructorInjectorStore.java:33)
	at com.google.inject.internal.FailableCache$1.load(FailableCache.java:48)
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3574)
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2316)
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2189)
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2079)
	... 45 more
Caused by: java.lang.NoClassDefFoundError: org/eclipse/xtext/testing/TemporaryFolder
	at java.base/java.lang.Class.getDeclaredFields0(Native Method)
	at java.base/java.lang.Class.privateGetDeclaredFields(Class.java:3297)
	at java.base/java.lang.Class.getDeclaredFields(Class.java:2371)
	at com.google.inject.internal.DeclaredMembers.getDeclaredFields(DeclaredMembers.java:40)
	at com.google.inject.spi.InjectionPoint.getDeclaredFields(InjectionPoint.java:809)
	at com.google.inject.spi.InjectionPoint.getInjectionPoints(InjectionPoint.java:719)
	at com.google.inject.spi.InjectionPoint.forInstanceMethodsAndFields(InjectionPoint.java:432)
	at com.google.inject.internal.ConstructorBindingImpl.getInternalDependencies(ConstructorBindingImpl.java:177)
	at com.google.inject.internal.InjectorImpl.getInternalDependencies(InjectorImpl.java:686)
	at com.google.inject.internal.InjectorImpl.cleanup(InjectorImpl.java:629)
	at com.google.inject.internal.InjectorImpl.initializeJitBinding(InjectorImpl.java:615)
	at com.google.inject.internal.InjectorImpl.createJustInTimeBinding(InjectorImpl.java:990)
	at com.google.inject.internal.InjectorImpl.createJustInTimeBindingRecursive(InjectorImpl.java:902)
	at com.google.inject.internal.InjectorImpl.getJustInTimeBinding(InjectorImpl.java:302)
	at com.google.inject.internal.InjectorImpl.getBindingOrThrow(InjectorImpl.java:225)
	at com.google.inject.internal.SingleFieldInjector.<init>(SingleFieldInjector.java:39)
	at com.google.inject.internal.MembersInjectorStore.getInjectors(MembersInjectorStore.java:131)
	at com.google.inject.internal.MembersInjectorStore.createWithListeners(MembersInjectorStore.java:97)
	at com.google.inject.internal.MembersInjectorStore.access$000(MembersInjectorStore.java:40)
	at com.google.inject.internal.MembersInjectorStore$1.create(MembersInjectorStore.java:49)
	at com.google.inject.internal.MembersInjectorStore$1.create(MembersInjectorStore.java:45)
	at com.google.inject.internal.FailableCache$1.load(FailableCache.java:48)
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3574)
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2316)
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2189)
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2079)
	... 60 more
Caused by: java.lang.ClassNotFoundException: org.eclipse.xtext.testing.TemporaryFolder
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	... 86 more


```

--- a/org.eclipse.xtext.xbase.testing/META-INF/MANIFEST.MF
+++ b/org.eclipse.xtext.xbase.testing/META-INF/MANIFEST.MF
@@ -10,7 +10,7 @@ Require-Bundle: org.eclipse.xtext;bundle-version="2.38.0",
  org.eclipse.xtext.xbase;bundle-version="2.38.0";visibility:=reexport,
  org.eclipse.jdt.core;bundle-version="3.37.0";visibility:=reexport,
  org.junit;bundle-version="4.13.2",
- org.eclipse.xtext.testing;bundle-version="2.38.0",
+ org.eclipse.xtext.testing;bundle-version="2.38.0";visibility:=reexport,
  org.eclipse.core.runtime;bundle-version="3.31.0"
 Bundle-ActivationPolicy: lazy
 Export-Package: org.eclipse.xtext.xbase.testing;version="2.38.0",

